$MirOS: ports/lang/egcs/gcc4.4/patches/patch-gcc_doc_invoke_texi,v 1.1 2014/02/10 01:15:51 tg Exp $
--- gcc/doc/invoke.texi.orig	Wed Mar 23 22:01:48 2011
+++ gcc/doc/invoke.texi	Thu May 29 19:55:53 2014
@@ -33,10 +33,10 @@ included in the gfdl(7) man page.
      funds for GNU development.
 @c man end
 @c Set file name and title for the man page.
-@setfilename gcc
+@setfilename gcc-VSS
 @settitle GNU project C and C++ compiler
 @c man begin SYNOPSIS
-gcc [@option{-c}|@option{-S}|@option{-E}] [@option{-std=}@var{standard}]
+gcc-VSS [@option{-c}|@option{-S}|@option{-E}] [@option{-std=}@var{standard}]
     [@option{-g}] [@option{-pg}] [@option{-O}@var{level}]
     [@option{-W}@var{warn}@dots{}] [@option{-pedantic}]
     [@option{-I}@var{dir}@dots{}] [@option{-L}@var{dir}@dots{}]
@@ -45,7 +45,7 @@ gcc [@option{-c}|@option{-S}|@option{-E}
     [@option{-o} @var{outfile}] [@@@var{file}] @var{infile}@dots{}
 
 Only the most useful options are listed here; see below for the
-remainder.  @samp{g++} accepts mostly the same options as @samp{gcc}.
+remainder.  @samp{g++-VSS} accepts mostly the same options as @samp{gcc-VSS}.
 @c man end
 @c man begin SEEALSO
 gpl(7), gfdl(7), fsf-funding(7),
@@ -234,7 +234,7 @@ Objective-C and Objective-C++ Dialects}.
 -Wconversion  -Wcoverage-mismatch  -Wno-deprecated  @gol
 -Wno-deprecated-declarations -Wdisabled-optimization  @gol
 -Wno-div-by-zero -Wempty-body  -Wenum-compare -Wno-endif-labels @gol
--Werror  -Werror=* @gol
+-Werror  -Werror=* -Werror-maybe-reset @gol
 -Wfatal-errors  -Wfloat-equal  -Wformat  -Wformat=2 @gol
 -Wno-format-contains-nul -Wno-format-extra-args -Wformat-nonliteral @gol
 -Wformat-security  -Wformat-y2k @gol
@@ -2671,6 +2671,11 @@ controls it, to determine what to use wi
 Note that specifying @option{-Werror=}@var{foo} automatically implies
 @option{-W}@var{foo}.  However, @option{-Wno-error=}@var{foo} does not
 imply anything.
+
+@item -Werror-maybe-reset
+@opindex Werror-maybe-reset
+Act like @samp{-Wno-error} if the @env{GCC_NO_WERROR} environment
+variable is set to anything other than 0 or empty.
 
 @item -Wfatal-errors
 @opindex Wfatal-errors
